app/src/main/java/com/example/myanimereport/models/ParseApplication.java

```diff
@@ -32,11 +32,12 @@ public void onCreate() {
         ParseObject.registerSubclass(BacklogItem.class);
         ParseObject.registerSubclass(Rejection.class);
         ParseObject.registerSubclass(AnimePair.class);
+        ParseObject.registerSubclass(UserGenre.class);
 
         // Initialize the parse application
         Parse.initialize(new Parse.Configuration.Builder(this)
-                .applicationId("OdPlemxtlEabuWLwikXOLhel5YXJWdzJNMDrrrVn")
-                .clientKey("twJERNUXhCGFVxzruL5KfoWwlGjgsKLFYlgS8ugJ")
+                .applicationId("k1vCItb2rteFbJOVnU6FmphLZWUDPn0jJDj16NH0")
+                .clientKey("cQxKUaYRw2Tz2Ba5BsAr2qWaxpZU9aLWjKq96pTv")
```

Comment:
We'll prob wanna move these to a secret file, instead of coding it into the app. Once thats done, there's a lot of documentation online on how to even remove it from the git history for maximum safety of keys ^